spec/models/agents/evernote_agent_spec.rb

```diff
@@ -0,0 +1,297 @@
+require 'spec_helper'
+
```

Comment:
I'm happy to do either. I haven't made a gem before, but I can give it a try. Any preferences?